config/environments/development.rb

```diff
@@ -57,4 +57,7 @@
       1, 32 * 1024 * 1024
     )
   )
+
+  # LOCAL change made specifically for CDRH
+  config.hosts << "cdrhdev1.unl.edu"
```

Comment:
I looked into the Rails documentation and source code to see if we could handle this with environment variables rather than in the config file, but looks like this is our only option. So I think this is the way to do it for all our Rails 6 apps for now.

Reply:
That's a bit inconvenient. :(